src/style.css

```diff
@@ -1 +1,70 @@
 /* Styles go here */
+* {
+  margin: 0;
+  padding: 0;
+}
```

Comment:
don't reset margin and padding in `*`tag

Reply:
understood